compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -365,12 +365,12 @@ OMR::X86::CodeGenerator::initialize(TR::Compilation *comp)
    self()->setSupportsArrayCmp();
    self()->setSupportsArrayCopy();
 
-   if (comp->getOption(TR_EnableX86AdvancedMemorySet))
-      self()->setSupportsArraySet();
-
    static char *disableArraySet = feGetEnv("TR_disableArraySet");
```

Comment:
This matches the p codegen so I think it is ok for now. It will at least allow us to determine if the evaluator is involved and then more debugging will be needed anyway and Victor would be the one doing that so since he made the PR and seemed happy I think we go with it.